### ISSUE TITLE
Blazor MAUI Android SSL Certification failed for self-signed server

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/runtime#63064]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/Blazor-MAUI-Android-SSL-Certification-fa/1597182)._

---
[severity:I'm unable to use this version]
I got error:

System.Net.WebException: 'java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.' site:stackoverflow.com

even I do like this:

// This method must be in a class in a platform project, even if
        // the HttpClient object is constructed in a shared project.
        public HttpClientHandler GetInsecureHandler()
        {
            HttpClientHandler handler = new HttpClientHandler();
            handler. ServerCertificateCustomValidationCallback = (message, cert, chain, errors) =>
            {
                //if (cert. Issuer.Equals("CN=localhost"))
                //    return true;
                //return errors == System.Net.Security.SslPolicyErrors.None;
                return true;
            };
            return handler;
        }

VS2022 preview with latest upgrade.

---
#### Feedback Bot on 11/28/2021, 10:54 PM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by Slm0nB** [comment]
Is this issue related to https://github.com/xamarin/xamarin-android/pull/6665 ?

**Comment by steveisok** [comment]
> Is this issue related to [xamarin/xamarin-android#6665](https://github.com/xamarin/xamarin-android/pull/6665) ?

Yes, this issue will be fixed by that PR.  @simonrozsival I assigned you to this issue so that we remember to close it when the PR lands.

**Comment by Eagle3386** [comment]
Sorry for the hurry, but as the only functional workaround replaces the whole trust manager currently:
Since the PR merged 2 days ago, when can we expect MAUI Blazor to include that fix?

**Comment by speschka** [comment]
I'm blocked by this right now as well, any info on how to implement a work around would be appreciated.

**Comment by kwyz** [comment]
Is this issue fixed, or what ?

**Comment by simonrozsival** [comment]
Yes, a bugfix has been released with MAUI GA. This issue should be closed, thanks for the ping @kwyz.
